source/docs/articles/drupal/cron.md

```diff
@@ -18,10 +18,7 @@ For every site environment, Pantheon executes cron at the top of each hour to al
 This bootstraps your site and invokes [drupal\_cron\_run](https://api.drupal.org/api/drupal/includes!common.inc/function/drupal_cron_run/7).
 
 There is no way to configure when Pantheon executes Drupal cron.
-<div class="alert alert-info" role="alert">
-<h4>Note</h4>
-Setting the value to "Never" will be ignored; cron will always run at least hourly.
-</div>
+
```

Comment:
The text in line 20 seems a little confusing given the FAQ on line 84
>  There is no way to configure when Pantheon executes Drupal cron.

Suggest a revision which states there's no a way to configure or disable Pantheon's system cron execution, but Drupal cron can be disabled from within `admin/system/config/cron` 

^ CSE and/or @newtoid should confirm that first though.